docs/examples/asyncagi/example.php

```diff
@@ -48,8 +47,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 // Mandatory stuff to bootstrap.
 ////////////////////////////////////////////////////////////////////////////////
-require_once 'PAMI/Autoloader/Autoloader.php'; // Include PAMI autoloader.
-\PAMI\Autoloader\Autoloader::register(); // Call autoloader register for PAMI autoloader.
+require_once 'autoload.php'; // Include Composer autoloader.
```

Comment:
Having an include path also tells PHP to look into other directories. Since you're using composer, that is not necessary any more.

Still, not a blocker for me :smile:.